Sources/ApexyLoader/ContentLoader.swift

```diff
@@ -1,3 +1,4 @@
+import Combine
```

Comment:
Do you need to add `#if canImport(Combine)`?

Reply:
Added.